0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -55,10 +55,10 @@ We assume that the sender knows the receiver endpoint (see [Appendix A: (Optiona
 1. The sender's SPSP client queries the receiver endpoint.
 2. The receiver endpoint responds with the receiver info, including the receiver's currency code.
 3. The sender chooses an amount of the receiver's asset to deliver.
-4. The sender's SPSP client submits the payment information, including the destination amount, to the receiver endpoint.
-5. The receiver endpoint responds with an [Interledger Payment Request](../0011-interledger-payment-request), which includes the execution condition.
-6. The sender's SPSP client uses its ILP module to get a quote in their currency or asset type for the ILP transfer.
-7. The sender accepts the quote.
+4. The sender's SPSP client uses its ILP module to get a quote in their currency or asset type for the ILP transfer.
+5. The sender accepts the quote.
+6. The sender's SPSP client submits the payment information, including the destination amount, to the receiver endpoint.
+7. The receiver endpoint responds with an [Interledger Payment Request](../0011-interledger-payment-request), which includes the execution condition.
```

Comment:
It's actually important at this step that the sender quote the payment request to make sure it matches what they thought it would be. It's kind of annoying to have two quotes in the flow but I'm not sure we can avoid it.